pkg/web/webhook.go

```diff
@@ -144,6 +163,43 @@ func processCheckRun(ctx context.Context, ds datastore.Datastore, checkAction, r
 	return nil
 }
 
+func receiveWorkflowJobWebhook(ctx context.Context, event *github.WorkflowJobEvent, ds datastore.Datastore) error {
+	action := event.GetAction()
+	installationID := event.GetInstallation().GetID()
+
+	repo := event.GetRepo()
+	repoName := repo.GetFullName()
+	repoURL := repo.GetHTMLURL()
+
+	labels := event.GetWorkflowJob().Labels
+	if !hasSelfHosted(labels) {
+		// workflow_job use GitHub-hosted, So will be ignored
+		logger.Logf(true, "self-hosted is not found in labels, so ignore (labels: %s)", labels)
+		return nil
+	}
+
+	if action != "queued" {
+		logger.Logf(true, "workflow_job actions is not queued, ignore")
+		return nil
+	}
+
+	jb, err := json.Marshal(event)
+	if err != nil {
+		return fmt.Errorf("failed to json.Marshal: %w", err)
+	}
+
+	return processCheckRun(ctx, ds, repoName, repoURL, installationID, jb)
+}
+
+func hasSelfHosted(labels []string) bool {
```

Comment:
FIXME

Reply:
fixed in https://github.com/whywaita/myshoes/pull/152/commits/79cf07acd6034f0500435a95f6b4984b1bc1caef